### PR TITLE
hw01_Mackenzie Whitfield 

### PR DESCRIPTION
**Pull request by mwhitfi1**
I edited the README to add my hobbies, places I've lived, educational background, and future goals. I found the video instructions incredibly helpful, clear, and easy to follow. I reached out to Dr. Soltoff via GitHub with a question about storing homework to a specific homework file. He answered my question quickly. The only issue I had was not being able to add images to the assignment. I reviewed the questions and answers started by my classmates on GitHub, but still had trouble. I ended up including a link rather than an image. By the time I did this, I realized I did not follow the 15 minute rule! I will make sure to do so in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+
+My name is Mackenzie Whitfield.  
+I moved to Chicago from San Diego two weeks ago. I **love** the beach. My hobbies also include hiking, traveling, reading, and watching movies. Up until now, I have only lived in California.  
+
+# Where I have Lived 
+1. San Diego  
+2. San Francisco  
+3. Los Angeles
+4. Chicago
+
```
Comment (ABacotti) [inline README.md:11]: I appreciate the use of the list function, I think it helps exemplify all the places that you have lived. I am sure that the weather here is much different than the places you have lived previously. I hope you are not too cold!

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+# Education
+I studied psychology at Loyola Marymount University in Los Angeles. I began participating in undergraduate research my freshman year and completed an Honors Thesis in my senior year. One of the *most* memorable experiences I had as an undergraduate was my study abroad in Greece. Below is a link to the study abroad page specific to my program. If you go under Student Testimonials, you will see the one I wrote! 
+[LMU Odyssey Program](https://bellarmine.lmu.edu/moderngreek/beyondtheclassroom/studyabroad)
+
```
Comment (ABacotti) [inline README.md:15]: This is a cool use of a link. I like how the text is different than the link address. It is a great way to provide more information about the topic that you are discussing, and it makes it easily accessible to the reader. I would love to see pictures of your trip!

```suggestion
 ![<alt text>](<insert path to image>)
```

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+# Future Plans  
+I am interested in a career in applied behavioral research, specifically in consumer psychology. While I do not know exactly where I might like to work, I look forward to spending my MAPSS year exploring my options, acquiring relevant skills, and networking with faculty and students.  
+
```
Comment (ABacotti) [inline README.md:18]: I really liked your work and enjoyed how you sectioned off each topic, maybe using differently sized headers could make it more aesthetically pleasing?

**Review by ABacotti — COMMENTED** [review]
I like your file! See in line comments

File: README.md [hunk 4]
```diff
@@ -3,13 +3,14 @@
...
 2. San Francisco  
 3. Los Angeles
+4. Chicago
 
 # Education
-I studied psychology at Loyola Marymount University in Los Angeles. I began participating in undergraduate research my freshman year and completed an Honors Thesis in my senior year. The experiences I had and the people I met over those for years were some of the best of my life. One of the *most* memorable was my study abroad in Greece. Below is a link to the study abroad page specific to this program. If you go under Student Testimonials, you will see the one I wrote! 
+I studied psychology at Loyola Marymount University in Los Angeles. I began participating in undergraduate research my freshman year and completed an Honors Thesis in my senior year. One of the *most* memorable experiences I had as an undergraduate was my study abroad in Greece. Below is a link to the study abroad page specific to my program. If you go under Student Testimonials, you will see the one I wrote! 
 [LMU Odyssey Program](https://bellarmine.lmu.edu/moderngreek/beyondtheclassroom/studyabroad)
 
 # Future Plans
```
Comment (rheuer83) [inline README.md:16]: I also really like the clarity of your homework. Adding one or two pictures to your work could help to make your contribution even more memorable!

**Review by rheuer83 — COMMENTED** [review]
Your homework is quite detailed and nicely organized. You have implemented many of the suggested markdown-elements and managed to submit the assignment using github. Nice job!

File: README.md [hunk 5]
```diff
@@ -1 +1,20 @@
 # hw01
+
+My name is Mackenzie Whitfield.  
+I moved to Chicago from San Diego two weeks ago. I **love** the beach. My hobbies also include hiking, traveling, reading, and watching movies. Up until now, I have only lived in California.  
+
+# Where I have Lived
```
Comment (minwenyang) [inline README.md:6]: The list function here is really clear and clean, a wonderful way of showing the cities!

File: README.md [hunk 6]
```diff
@@ -1 +1,20 @@
...
+My name is Mackenzie Whitfield.  
+I moved to Chicago from San Diego two weeks ago. I **love** the beach. My hobbies also include hiking, traveling, reading, and watching movies. Up until now, I have only lived in California.  
+
+# Where I have Lived 
+1. San Diego  
+2. San Francisco  
+3. Los Angeles
+4. Chicago
+
+# Education
```
Comment (minwenyang) [inline README.md:12]: I really like the use of headers, it makes the text really organized. And it looks like a really interesting experience traveling to Greece, looking forward to your photos!

File: README.md (hunk 3 above)
Comment (minwenyang) [inline README.md:18]: Really interesting future plans! I am also interested in consumer behavior, so maybe we can chat about it afterward!

**Comment by minwenyang** [comment]
Mackenzie used headers, lists, emphasis, and links in the homework, which fulfilled the markdown requirements. The reporting of the process and the reflection are very clear. I really enjoyed this fun and organized self-introduction!

**Review by Donghyun-Kang-Soc — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |     X         |            |
| **Coding strategy**      |           |      X        |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |           |     X         |            |
| **Reproducibility**         |     X      |              |            |

Remarks:
* Thanks for providing a detailed introduction with the R markdown syntax.
* Please try to use different elements like an unordered list and other headers' levels next time you submit your assignment.

**Pull request closed without merging**